dotcom-rendering/src/model/extractTrendingTopics.ts

```diff
@@ -49,23 +49,23 @@ const dedupeTags = (tag: FETagType[]): FETagType[] =>
 		.map((id) => tag.find((item) => item.properties.id === id))
 		.filter(isNonNullable);
 
-export const extractTrendingTopics = (
+export const extractTrendingTopicsFomFront = (
 	collections: FECollectionType[],
 ): FETagType[] => {
 	// Get a single array of all trails in the collections
+
 	const allTrails = collections.flatMap((collection) => [
 		...collection.curated,
 		...collection.backfill,
 	]);
 
+	return extractTrendingTopics(allTrails);
+};
+export const extractTrendingTopics = (trails: FEFrontCard[]): FETagType[] => {
 	// Remove any duplicated trails
-	const dedupeTrails = (trails: FEFrontCard[]): FEFrontCard[] =>
-		trails.filter((trailOne) =>
-			trails.findIndex(
-				(trailTwo) => trailOne.card.id === trailTwo.card.id,
-			),
-		);
-	const dedupedTrails = dedupeTrails(allTrails);
+	const dedupedTrails = trails.filter((trailOne) =>
+		trails.findIndex((trailTwo) => trailOne.card.id === trailTwo.card.id),
```

Comment:
I think I will leave this as-is for this PR since I'm only really moving code around in this file, but I'm sure there's some refactors available in this code area for future PRs :)